postgres/seed.js

```diff
@@ -0,0 +1,27 @@
+const { createTables, dropTables } = require("./migrations/createTables.js");
```

Comment:
Sounds like you may not use postgres, you may want to remove this in your next PR